source/usage-examples/findOne.txt

```diff
@@ -4,100 +4,51 @@ Find a Document
+   :start-after: begin findOne
+   :end-before: end findOne
+   :language: go
+   :dedent:
+   :emphasize-lines: 3
 
-- ``nil`` if a document matched your query, and there were no errors retrieving
-  the document.
-- ``ErrNoDocuments`` if no documents matched your query.
-- If the driver retrieved your document but could not unmarshal your result, the
-  ``Decode()`` method returns the unmarshaling error.
-- If there was an error retrieving your document during execution of the
-  ``FindOne()`` method, the error propagates to the ``Decode()`` method and
-  the ``Decode()`` method returns the error.
+Click `here <{+example+}/findOne.go>`__ to see a fully runnable example.
 
-Example
--------
+Expected Result
+---------------
 
-The following code example retrieves a single document from the ``movies``
-collection. It uses the following arguments for the ``FindOne()`` method:
+After you run the code example, you will see output similar to:
 
-- An empty **Context** instance returned from the ``context.TODO()`` function.
-- A **filter document** that only matches movies with the title exactly matching
-  the text ``"The Room"``.
-- **Options** that specify a **sort** of matched documents in
-  descending order by rating, such that the first document has the highest
-  rating. The options also specify a **projection** for the results so
-  that the MongoDB instance only sends the ``title`` and ``imdb`` fields of
-  the retrieved document.
+.. code-block:: json
+   :copyable: False
 
-.. literalinclude:: findOne.go
-   :language: go
+   {
+     ...
+     "title": "The Room",
+     "year": 2003,
+     ...
+   }
 
-If you run the example above, you should see a result that looks like this:
+Additional Information
+----------------------
 
-.. literalinclude:: findOne-output.txt
-   :language: none
+For more information on specifying query filters and
+handling potential errors, see our guide on **<TODO: retrieve data
+fundamental page>**.
 
-.. include:: /includes/sample-data-note.rst
+For more information on query operators,
+see the :manual:`MongoDB query operator reference documentation
+</reference/operator/query/>`.
 
-For more information on the methods, interfaces, structs, and functions
-mentioned on this page, see the following API documentation:
+API Documentation
+~~~~~~~~~~~~~~~~~
 
-- :go-api:`Collection.FindOne() <mongo#Collection.FindOne>`
-- `Context <https://pkg.go.dev/context#Context>`__
-- :go-api:`FindOneOptions <mongo/options#FindOneOptions>`
-- :go-api:`FindOneOptions.SetProjection() <mongo/options#FindOneOptions.SetProjection>`
-- :go-api:`options.FindOne() <mongo/options#FindOne>`
-- :go-api:`SingleResult <mongo#SingleResult>`
-- :go-api:`SingleResult.Decode() <mongo#SingleResult.Decode>`
-- `context.TODO() <https://pkg.go.dev/context#TODO>`__
-- :go-api:`bson.D <bson#D>`
-- :go-api:`bson.MarshalExtJSON() <bson#MarshalExtJSON>`
+- `FindOne() <{+godocs+}/mongo#Collection.FindOne>`__
```

Comment:
Nit/Question: If there's only one item, should there be a bullet? Other pages with only one currently don't.